common/libexec/wwinit/10-database.init

```diff
@@ -90,45 +115,35 @@ fi
 
 if [ -z "$DBVERS" ]; then
     wwprint "Creating database schema"
-    if echo "$SCHEMA" | mysql "$DBNAME"; then
+    if echo "$SCHEMA" | mysql $DBOPTIONS "$DBNAME"; then
```

Comment:
This should probably be done via a redirect so we don't loose formatting/quoting/escaping through the echo.